app/api/managers.py

```diff
@@ -0,0 +1,33 @@
+from collections import Counter
+
+from django.db.models import Manager, Count
+
+
+class AnnotationManager(Manager):
```

Comment:
Interesting. I didn't know about managers. Curious what was the thought process on deciding for a manager instead of an instance method on the model object?